atlp-rwanda/dynamites-ecomm-fe
# Implement Customer Management

### What does this PR do?
This PR implements the customer management feature for the admin.

### Description of Task to be completed?

- Implement customer management UI components
- Integrate components with the backend
- Add CRUD operations for customer management
- Write appropriate tests

### How should this be manually tested?

- Log in as an admin.
- Navigate to the customer management page.
- Perform CRUD operations (Create, Read, Update, Delete) on customers.
- Ensure the page is responsive and works on different devices.

### Any background context you want to provide?
This feature allows admins to manage customer information efficiently through a dedicated customer management interface. It includes full CRUD operations and integrates with the backend to ensure data persistence.

### What are the relevant pivotal tracker/Trello stories?
NA

### Screenshots (if appropriate)

![image](https://github.com/user-attachments/assets/64633882-84e9-4655-bf62-9681ddc1c665)

![Screenshot 2024-07-24 191828](https://github.com/user-attachments/assets/8be7575d-c554-487e-8d34-8478bea714b0)

![Screenshot 2024-07-24 191815](https://github.com/user-attachments/assets/b2d6a547-9123-4672-b814-804460b052ab)

![Screenshot 2024-07-24 191759](https://github.com/user-attachments/assets/fdf71351-9c9c-4a75-a1a9-39f0259ebf69)


### Questions:
NA
